module/config.toml

```diff
@@ -0,0 +1,40 @@
+[module]
+_merge = "deep"
+# work around https://github.com/golang/go/issues/37397
+[[module.mounts]]
+  source = "assets/bootstrap/scss/_vendor"
+  target = "assets/vendor/bootstrap/scss/vendor"
+# mount user guide so that we can copy it to skeleton site
+[[module.mounts]]
+  source = "userguide"
+  target = "assets/userguide"

```

Comment:
Do we ever copy the user guide to a skeleton site? Is this a documented "feature"?

Otherwise, can we drop the `userguide` mount?

Reply:
> Do we ever copy the user guide to a skeleton site?

No. But I saw that other hugo themes are doing this and I liked it.

> Is this a documented "feature"?

No not yet. But I have some ideas how to make installation process easier. I will start a discussion on this shortly. If we agree upon that copying over the user guide is a good thing to do, I'm willing to document this.

> Otherwise, can we drop the `userguide` mount?

We can drop it and readd it later.

Reply:
So I think we didn't consider automating pushing the user guide to a skeleton site because we already have docsy-example (we do suggest users can manually copy it if the structure suits them better than docsy-example - it's smaller, for one). Might be one to ask the community if it'd be useful or just kind of redundant? 

(we *could* consider making the user guide the example site but then you don't get all the various boilerplate bits of text/structure we've put into it)

Reply:
Right. I'd rather we continue to use the `docsy-example` as a starter/base instead of repurposing the user guide.

Reply:
Mount was removed in e444964a013e.